src/ProxyManager/Generator/ClassGenerator.php

```diff
@@ -33,7 +33,7 @@ class ClassGenerator extends ZendClassGenerator
     /**
      * {@inheritDoc}
      */
-    public function setExtendedClass($extendedClass) : self
+    public function setExtendedClass($extendedClass) : parent
```

Comment:
That's just the `Zend\Code` API, which is fluent. I can't change the API in a subclass